### ISSUE TITLE
Is this still maintained by you?

### ISSUE THREAD

**Issue by M-Reimer** [mdre77/vdr-plugin-vnsiserver#5]
There have been pull requests in the past where you never responded.

A few days ago one developer, who wants to add some features to vnsiserver, asked if the plugin could go "community maintained" as there currently seems to be no active maintainer.

To handle such "unmaintained" VDR projects, some people of the VDR community (including me) have created a GitHub organization called "vdr-projects" where projects can be moved to if they get unmaintained to provide some target for developers to point pull requets to: https://vdr-projects.github.io/

So long story short: Do you still plan to actively do development to the vnsiserver plugin or should it be moved to a "community maintained" project where individual contributors can easier contribute to?

**Comment by M-Reimer** [comment]
No answer since one month so I guess this repo is abandoned. To make it possible to do further development, I've created a repo here: https://github.com/vdr-projects/vdr-plugin-vnsiserver
